web/pdf_sidebar.js

```diff
@@ -104,7 +106,7 @@ var PDFSidebar = (function PDFSidebarClosure() {
     reset: function PDFSidebar_reset() {
       this.isInitialViewSet = false;
 
-      this.close();
+      this._hideUINotification(null);
```

Comment:
This is just fallout from testing, since I wanted to keep the sidebar open to be able to check that all notifications were being removed on reset as intended.

However, I'm not sure that we actually *must* close the sidebar when resetting it; so I'm not sure if we need to but the `this.close();` call back!?